code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -107,11 +107,15 @@ This file's folder contains:
 			var/mob/living/silicon/robot/R = S
 			R.UnlinkSelf()
 			R.emagged = 1
-			R << "<span class='warning'><b>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</b></span>"
+			R << "<span class='boldwarning'>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</span>"
 		S.laws = new/datum/ai_laws/ratvar
 		S.laws.associate(S)
 		S.update_icons()
 		S.show_laws()
+		var/datum/action/innate/hierophant/H = new()
+		H.Grant(S)
+		H.title = null //so it's just the borg's name
+		S << "<span class='heavy_brass'>You can communicate with other servants by using the Hierophant Network action button in the upper left.</span>"
```

Comment:
I accidentally misspelled 'span' as 'psna' except it actually worked, ie, properly applied the span class, for some reason.